node-src/tasks/report.ts

```diff
@@ -1,11 +1,28 @@
-import reportBuilder from 'junit-report-builder';
+import reportBuilder, { TestSuite } from 'junit-report-builder';
 import path from 'path';
 
 import { createTask, transitionTo } from '../lib/tasks';
 import { Context } from '../types';
 import wroteReport from '../ui/messages/info/wroteReport';
 import { initial, pending, success } from '../ui/tasks/report';
 
+interface TestMode {
+  name: string;
+}
+
+interface TestSpec {
+  name: string;
+  component: {
+    name: string;
+    displayName: string;
+  };
+}
+
+interface TestParameters {
+  viewport: number;
+  viewportIsDefault: boolean;
+}
+
```

Comment:
Should we integrate these types into `ReportQueryResult` so we're not repeating them?